packages/adblocker/src/events.ts

```diff
@@ -87,37 +87,52 @@ function triggerCallback<EventNames>(
  * Node.js) allowing partially typed event emitting. The set of event names is
  * specified as a type parameter while instantiating the event emitter.
  */
-export class EventEmitter<EventNames> {
+export class EventEmitter<
```

Comment:
changes to this file are completely optional, right?

Reply:
It is, but this is required for the internal type enforcement.